packages/remix/index.ts

```diff
@@ -1,6 +1,9 @@
 // This class exists to prevent https://github.com/remix-run/remix/issues/2031 from occuring
 export class RemixNotSetupError extends Error {
   constructor() {
+    console.warn(
+      "DEPRECATION: Importing from `remix` is deprecated. Import from `@remix-run/*` packages instead."
```

Comment:
This occurrence is outside of `remix-dev` package, so can't use our nice `deprecation(...)` utility here.